packages/iris-grid/src/IrisGridDataBarCellRenderer.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  BoxCoordinates,
+  Coordinate,
+  DataBarCellRenderer,
+} from '@deephaven/grid';
+import {
+  vsTriangleDown,
+  vsTriangleRight,
+  IconDefinition,
+} from '@deephaven/icons';
+import { IrisGridRenderState } from './IrisGridRenderer';
+
+const ICON_SIZE = 16;
+const ICON_NAMES = Object.freeze({
+  CARET_DOWN: 'caretDown',
+  CARET_RIGHT: 'caretRight',
+});
```

Comment:
This icon stuff is now copied in multiple places, let's refactor it so we can use it from the same places. In fact, we can actually cache the same icon across all instances of grids (right now we have separate icons cached for each grid, and even each cell renderer).
This code was also written in old JS times, we can take advantage of TypeScript to make it more well defines.
Create a new file called `IrisGridIcons.ts`, and add something like the following:
```
export const ICON_SIZE = 16;

export type IconName = 'sortUp'  | 'sortDown' | 'caretDown' | 'caretRight' | 'cellOverflow';

const iconMap = new Map<IconName, IconDefinition>([
  ['sortUp', dhSortUp],
  ...
]);

const makeIcon = memoize((name: IconName) => {
    const path = Array.isArray(faIcon.icon[4])
      ? faIcon.icon[4][0]
      : faIcon.icon[4];
    const icon = new Path2D(path);
    const scaledIcon = new Path2D();
    const scaleMatrix = {
      a: ICON_SIZE / faIcon.icon[0],
      d: ICON_SIZE / faIcon.icon[1],
    };
    scaledIcon.addPath(icon, scaleMatrix);
   return scaledIcon
}, { max: 1000 });

export function getIcon(name: IconName): Path2D {
  return makeIcon(name);  
}
```

Then get rid of all the icons stuff from inside the renderers, and just do something like:
```
import { getIcon } from './IrisGridIcons';

...
getIcon('caretDown')
```